print-battleground-state-changes

```diff
@@ -377,7 +383,7 @@ def json_to_summary(
         candidate2_votes,
         vote_diff,
         votes_remaining,
-        new_votes,
+        f"{new_votes_relevant:7,}" if new_votes_relevant >= 0 else "Unknown",
```

Comment:
This means the CSV will include commas (or the string `Unknown`). I don't have strong opinions on this, just noting it for other reviewers.

Reply:
Oof, good call out! Python's csv library would quote it correctly, but it wouldn't be a trivial number anymore.

Let me think about how to handle this.

Reply:
Lets just add a new column w/ a new name. That should minimize the risk of breaking anyone's existing data handling.